utkdigitalinitiatives/utk_isl_xml_forms
# Address TRAC-635.

**JIRA Ticket**: [TRAC-635](https://jira.lib.utk.edu/browse/TRAC-635)

# What does this Pull Request do?

Removes supplemental file description.

# What's new?

This simply removes the supplemental file section of the form. No intended consequences, but this is XML Forms we're talking here.

# How should this be tested?

1. Option 1:
	* Import the Form
	* Associate it with a content model
	* Apply any additional / related transforms
	* Create a new record and select the newly associated form
	* Edit that record to see if the form still behaves correctly
        * No elements should be added instead of updated.

# Interested parties
@CanOfBees @DonRichards
